src/pages/settings/Payments/TransferBalancePage.js

```diff
@@ -0,0 +1,249 @@
+import HeaderWithCloseButton from '../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../components/ScreenWrapper';
+import Navigation from '../../../libs/Navigation/Navigation';
+import styles from '../../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import compose from '../../../libs/compose';
+import KeyboardAvoidingView from '../../../components/KeyboardAvoidingView/index';
+import * as Expensicons from '../../../components/Icon/Expensicons';
+import MenuItem from '../../../components/MenuItem';
+import CONST from '../../../CONST';
+import variables from '../../../styles/variables';
+import ExpensifyText from '../../../components/ExpensifyText';
+import ExpensifyButton from '../../../components/ExpensifyButton';
+import FixedFooter from '../../../components/FixedFooter';
+import CurrentWalletBalance from '../../../components/CurrentWalletBalance';
+import * as paymentPropTypes from './paymentPropTypes';
+import * as PaymentMethods from '../../../libs/actions/PaymentMethods';
+import * as PaymentUtils from '../../../libs/PaymentUtils';
+import getBankIcon from '../../../components/Icon/BankIcons';
+
+const propTypes = {
+    /** User's wallet information */
+    userWallet: paymentPropTypes.userWalletPropTypes,
+
+    /** Array of bank account objects */
+    bankAccountList: paymentPropTypes.bankAccountListPropTypes,
+
+    /** Array of card objects */
+    cardList: paymentPropTypes.cardListPropTypes,
+
+    /** Wallet balance transfer props */
+    walletTransfer: paymentPropTypes.walletTransferPropTypes,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    userWallet: {
+        walletLinkedAccount: {},
+    },
+    bankAccountList: [],
+    cardList: [],
+    walletTransfer: {},
+};
+const Fee = 0.30;
+
+class TransferBalancePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.paymentTypes = [
+            {
+                key: CONST.WALLET.TRANSFER_METHOD_TYPE.INSTANT,
+                title: this.props.translate('transferAmountPage.instant'),
+                description: this.props.translate('transferAmountPage.instantSummary', {
+                    amount: this.props.numberFormat(
+                        0.25,
+                        {style: 'currency', currency: 'USD'},
+                    ),
+                }),
+                icon: Expensicons.Bolt,
+                type: CONST.PAYMENT_METHODS.DEBIT_CARD,
+            },
+            {
+                key: CONST.WALLET.TRANSFER_METHOD_TYPE.ACH,
+                title: this.props.translate('transferAmountPage.ach'),
+                description: this.props.translate('transferAmountPage.achSummary'),
+                icon: Expensicons.Bank,
+                type: CONST.PAYMENT_METHODS.BANK_ACCOUNT,
+            },
+        ];
+        PaymentMethods.startWalletTransfer(this.props.userWallet.currentBalance - Fee);
```

Comment:
What does it mean to "start" a wallet transfer?

Reply:
We have to set some data that is necessary to manage the component state before users interact with anything on Wallet TransferPage. What about renaming it `setWalletTransferInitialData`

Reply:
Can we try to think of a name that communicates _why_ we need to do this? Which state needs to be reset and how are we using it?

Reply:
Oh yeah, `setTransferAmountAndSelectedAccount`. the three principles.